bears/shell/ShellCheckBear.py

```diff
@@ -20,8 +20,13 @@ class ShellCheckBear:
     CAN_DETECT = {'Syntax', 'Security', 'Undefined Element', 'Unused Code'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file, shell: str='sh'):
+    def create_arguments(filename, file, config_file, shell: str='sh',
+                         ignore_rules: list=[]):
         """
         :param shell: Target shell being used.
         """
```

Comment:
what about ``:param ignore_rules:``, you have to add the docs for this setting too